Sources/Orbit/Components/EmptyState.swift

```diff
@@ -65,30 +61,52 @@ public extension EmptyState {
     init(
         _ title: String = "",
         description: String = "",
-        illustration: Illustration.Image,
-        action: Action = .none
+        illustration: Illustration.Image = .none,
+        @ViewBuilder content: () -> Content = { EmptyView() },
+        @ButtonStackBuilder buttons: () -> Buttons
     ) {
-        self.title = title
-        self.description = description
-        self.illustration = illustration
-        self.action = action
+        self.init(
+            title: title,
+            description: description,
+            illustration: illustration
+        ) {
+            content()
+        } buttons: {
+            buttons()
+        }
+    }
+
+    /// Creates Orbit EmptyState component with no action.
+    init(
+        _ title: String = "",
+        description: String = "",
+        illustration: Illustration.Image = .none
+    ) where Content == EmptyView, Buttons == EmptyView {
```

Comment:
Can we use a default value for `Buttons` in the init above instead? Or is that a problem because of the order of closures? If so, shouldn't buttons be first?

Reply:
Yep, the order of closures is the issue - even more that the Button when used in Content does not work with .status() modifier, but instead silently uses the Button as-is, as the custom content.

Another option would be to remove the custom content support completely, but this removes the flexibility for customization.